Gopkg.toml

```diff
@@ -63,7 +63,7 @@ required = [
 
 [[constraint]]
   name = "github.com/jcrossley3/manifestival"
-  branch = "master"
+  branch = "newideas"
```

Comment:
Before merging this, do you anticipate merging the `newideas` manifestival branch into its master?

Reply:
Yes